packages/flutter/lib/src/material/app.dart

```diff
@@ -472,6 +476,28 @@ class MaterialApp extends StatefulWidget {
   ///    system what kind of theme is being used.
   final ThemeMode? themeMode;
 
+  /// The duration of animated theme changes.
+  ///
+  /// When the theme changes (either by the [theme], [darkTheme] or [themeMode]
+  /// parameters changing) it is animated to the new theme over time.
+  /// [themeAnimationDuration] determines how long this animation takes.
+  ///
+  /// To have the theme change immediately, you can set this to [Duration.zero].
+  ///
+  /// The default it is [kThemeAnimationDuration].
```

Comment:
nit: this sentence reads awkward. Maybe remove the "it"?

Reply:
D'oh! I english'd good. Thx.